registry/extension/oras/artifactservice.go

```diff
@@ -50,17 +55,27 @@ func (h *referrersHandler) Referrers(ctx context.Context, revision digest.Digest
 			return nil
 		}
 
-		desc, err := blobStatter.Stat(ctx, referrerRevision)
-		if err != nil {
-			return err
+		// filtering by artifact type
+		if ArtifactMan.ArtifactType() == referrerType {
```

Comment:
Save to variable since it's used in multiple places

Reply:
switched to a variable